cylc/flow/xtrigger_mgr.py

```diff
@@ -318,13 +321,14 @@ def validate_xtrigger(
     def add_trig(self, label: str, fctx: 'SubFuncContext', fdir: str) -> None:
         """Add a new xtrigger function.
 
-        Check the xtrigger function exists here (e.g. during validation).
+        Call check_xtrigger before this, during validation.
+
         Args:
             label: xtrigger label
             fctx: function context
             fdir: function module directory
+
         """
-        self.validate_xtrigger(label, fctx, fdir)
```

Comment:
This causes the check to no longer happen here at run time:
https://github.com/cylc/cylc-flow/blob/2538bcf38da38f0267d05a9590ebc91888e0c27b/cylc/flow/config.py#L2432-L2435

Reply:
I don't think it needs to happen at run time. The args are all pre-determined.

Reply:
(Ideally the xtrigger_mgr would exist for validation-only too, but a bit of refactoring is required to achieve that, because of all the other managers currently need to initialize it).